app/editor/src/features/admin/ingests/ContentReferenceList.tsx

```diff
@@ -127,6 +142,17 @@ const ContentReferenceList: React.FC<IContentReferenceListProps> = (props) => {
             setFilter({ ...filter, uid: e.target.value });
           }}
         />
+        <Select
+          label="Source"
+          name="sources"
+          options={sourceOptions}
+          onChange={(newValue: any) => {
+            // Use the source.code to set the Kafka topic.
```

Comment:
could this potentially break TNO migrations?  They all share a common source, but the topics are all different?

Reply:
Before it would filter on all possible sources.  I think this will behave the same way for TNO.